app/code/Magento/ConfigurableProduct/Test/Unit/Ui/DataProvider/Product/ConfigurableDataProviderTest.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ConfigurableProduct\Test\Unit\Ui\DataProvider\Product;
+
+use Magento\Catalog\Model\ResourceModel\Product\Collection;
+use Magento\Catalog\Model\ResourceModel\Product\CollectionFactory;
+use Magento\ConfigurableProduct\Ui\DataProvider\Product\ConfigurableDataProvider;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Ui\DataProvider\Modifier\PoolInterface;
+use Magento\Ui\DataProvider\Modifier\ModifierInterface;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * ConfigurableDataProviderTest class which checks type
+ */
+class ConfigurableDataProviderTest extends TestCase
+{
+    /**
+     * @var ObjectManager
+     */
+    protected $objectManager;
```

Comment:
All protected should be private

Reply:
Fixed by changing all protected to private